src/pytorch_lightning/lite/wrappers.py

```diff
@@ -56,21 +52,20 @@ def optimizer(self) -> Optimizer:
         return self._optimizer
 
     def state_dict(self) -> Dict[str, Tensor]:
-        return self._strategy.optimizer_state(self.optimizer)
+        return self._strategy.get_optimizer_state(self.optimizer)
 
-    def step(self, closure: Optional[Callable] = None) -> Any:
-        closure = closure or _do_nothing_closure
+    def step(self, closure: Optional[Callable] = None, module: Optional["_LiteModule"] = None) -> Any:
```

Comment:
Why do you add this argument? For deepspeed users using Lite?

Reply:
Yes. It could stay or go, either way it does not make a big difference.

Reply:
I'd remove it unless there's a real worthy usecase for it